test/Broadway/Domain/DateTimeTest.php

```diff
@@ -28,12 +28,23 @@ public function it_converts_back_and_forth()
 
     /**
      * @test
+     * @dataProvider providesLocales
      */
-    public function it_creates_now()
+    public function it_creates_now($locale)
     {
+        $this->setLocale(LC_ALL, $locale);
```

Comment:
Don't think it matters that much, but we could use LC_TIME here.
But as it resets anyway, I'm fine with using LC_ALL here.